modules/seqexec/server/src/main/scala/seqexec/server/EpicsUtil.scala

```diff
@@ -124,33 +90,18 @@ trait EpicsSystem[T] {
 
 object EpicsCommand {
 
-  sealed trait Result
-  object Paused extends Result
-  object Completed extends Result
-
-  def safe[A](a: SeqAction[A]): SeqAction[A] = EitherT(a.value.attempt.map {
-    case Left(e)  => SeqexecException(e).asLeft
-    case Right(r) => r
-  })
-
-  def setParameter[A, T](p: Option[CaParameter[T]], v: A, f: A => T): SeqAction[Unit] =
-    safe(SeqAction.either {
-      p.map(_.set(f(v)).asRight).getOrElse(SeqexecFailure.Unexpected("Unable to set parameter.").asLeft)
-    } )
-
-  def setParameter[T](p: Option[CaParameter[T]], v: T): SeqAction[Unit] =
-    safe(SeqAction.either {
-      p.map(_.set(v).asRight).getOrElse(SeqexecFailure.Unexpected("Unable to set parameter.").asLeft)
-    } )
+  sealed trait Result extends Product with Serializable
+  case object Paused extends Result
+  case object Completed extends Result
```

Comment:
I think elsewhere we're using a style where instances are defined inside the companion, so `Result.Paused`, etc.

Reply:
yes, I have been changing some but I missed this one